packaging/fedora/upstream.spec

```diff
@@ -43,7 +43,7 @@ Requires:      %{name}
 Requires:      python3
 Requires:      python%{python3_pkgversion}-lark-parser
 Requires:      python%{python3_pkgversion}-requests
-Provides:      bundled(python%{python3_pkgversion}-lark-parser) = 0.11.3
+Provides:      bundled(python%{python3_pkgversion}-lark-parser) = 1.1.8
```

Comment:
I don't know what this file is about — something related to Jordan’s out-of-tree RPM packaging — and it looks pretty stale, so I think this PR should probably not change it. `charliecloud.spec` is the one we test.